sklearn/impute/_knn.py

```diff
@@ -90,10 +90,10 @@ class KNNImputer(_BaseImputer):
 
     See Also
     --------
-    SimpleImputer : Imputation transformer for completing missing values
+    SimpleImputer : Univariate imputer for completing missing values
         with simple strategies.
-    IterativeImputer : Multivariate imputer that estimates each feature
-        from all the others.
+    IterativeImputer : Multivariate imputer that estimates values to impute for
+        each feature with missing values from all the others.
```

Comment:
For the sake of consistency, please make the same change in the first line of the `IterativeImputer` docstring and other references to it in `sklearn/impute/_base.py`

Reply:
Since it does not fit on a single line, it will not be compliant with `numpydoc`. I think that we can let it as-is